main.go

```diff
@@ -59,10 +60,12 @@ func main() {
 	var watchNamespaces string
 	var excludeNamespaces string
 	var recordChanges bool
+	var secretTTL time.Duration
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.DurationVar(&reconcilePeriod, "reconcile-period", 5*time.Second, "How often the controller will re-queue vals-operator events.")
+	flag.DurationVar(&secretTTL, "ttl", 300*time.Second, "How often to check backend for updates.")
```

Comment:
I'm inclined to rename this to "defaultTTL" or something like that because it can be overridden per secret. The help should also be updated to reflect that this is a default